src/helpers/PromptsHelper.ts

```diff
@@ -0,0 +1,14 @@
+export default class PromptsHelper {
+  static isCategorySlidedownConfigured(): boolean {
+    const { promptOptions } = OneSignal.context.appConfig.userConfig;
+    if (!promptOptions) {
+      return false;
+    }
+
+    const isUsingCategoryOptions = !!promptOptions && !!promptOptions.slidedown && !!promptOptions.slidedown.categories;
+    return (
+      isUsingCategoryOptions &&
+      !!promptOptions.slidedown!.categories!.tags &&
```

Comment:
1. It's not recommended to silence typescript errors. We usually do this only in tests and only if necessary. The code should be rewritten to avoid it. Example:
```
static isCategorySlidedownConfigured(context: ContextInterface): boolean {
    const { promptOptions } = context.appConfig.userConfig;

    if (!promptOptions || !promptOptions.slidedown || !promptOptions.slidedown.categories) {
      return false;
    }

    return (!!promptOptions.slidedown.categories.tags && promptOptions.slidedown.categories.tags.length > 0);
  }
```
2. we want to stop using global accessor for context if possible. It can easily be done with passing it as a param to this helper method.
It would also require changing EventsHelper like this:
```
private static async onSubscriptionChanged_sendCategorySlidedownTags(isSubscribed: boolean | undefined) {
    if (isSubscribed !== true) {
      return;
    }

    const context = OneSignal.context;

    if (PromptsHelper.isCategorySlidedownConfigured(context)) {
      await context.tagManager.sendTags(false);
    }
  }
```
and PromptsManager which already has reference to the context.

Reply:
Done.